db/dependents/dependents_utils.py

```diff
@@ -133,16 +183,25 @@ def _get_typed_dependency_pairs_stmt(engine):
     pg_identify_refobject = _get_pg_identify_object_lateral_stmt(
         pg_depend.c.refclassid, pg_depend.c.refobjid, 0)
     pg_constraint = _get_pg_constraint_table(engine, metadata)
+    pg_rewrite = _get_pg_rewrite(engine, metadata)
 
     # each statement filters the base statement extracting dependents of a specific type
     # so it's easy to exclude particular types or add new
     dependency_pairs = _get_dependency_pairs_stmt(pg_depend, pg_identify_object, pg_identify_refobject)
     foreign_key_constraint_dependents = _get_foreign_key_constraint_dependents(pg_identify_object, dependency_pairs).cte('foreign_key_constraint_dependents')
     table_dependents = _get_table_dependents(foreign_key_constraint_dependents, pg_constraint).cte('table_dependents')
 
+    # should not be returned directly, used for getting views
+    rule_dependents = _get_rule_dependents(pg_identify_object, dependency_pairs).cte('rule_dependents')
```

Comment:
Why do we need to use rule dependencies to find view dependencies? Can you add a comment on why this is needed